internal/drivers/google/option.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type Option func(*provider)
+type Option func(*googleConfig)
```

Comment:
there's no need for this to be googleConfig - you're already in the package "google" so "config" is sufficient

Reply:
same for the rest of them.